feature-group-definitions/font-palette-animation.yml

```diff
@@ -0,0 +1,5 @@
+name: font-palette animation
+spec: https://drafts.csswg.org/css-fonts-4/#font-palette-prop
+usage_stats: https://chromestatus.com/metrics/css/timeline/animated/709
```

Comment:
It's not clear to me that this usage counter belongs here, duplicating the other feature. Can you elaborate on this?

Reply:
It's not the same URL, this is for when `font-palette` is animated.

Reply:
Ah, this was a copy-and-paste failure on my part (I opened "both). Sorry!